.yamato/config/__shared.metafile

```diff
@@ -1,21 +1,11 @@
-target_branch: master # specifies the branch on which recurrent or pr triggers are created, etc
-target_editor: trunk # specifies the target editor used for some package/preview publish jobs
+target_branch: 2021.2/staging # specifies the branch on which recurrent or pr triggers are created, etc
+target_editor: 2021.2 # specifies the target editor used for some package/preview publish jobs
 conditional_jobs: # some jobs are relevent only depending on the branch
   preview_publish: True
   formatting: True
   packages.package_publish_all: True
   packages.project_publish_all: True
 editors: # applied for all yml files
-  - track: trunk
-    name: trunk
-    rerun_strategy: on_new_revision
-    ono_branch_var: ONO_BRANCH_TRUNK
-    editor_pinning:
-      enabled: True
-      nr_commits_to_retrieve: 1
```

Comment:
Same/opposite as with the other PR, I think we should set `nr_commits_to_retrieve: 1` for 2021.2, since it should be more stable now and we don't need bisection on it?